songpal/device.py

```diff
@@ -464,7 +465,7 @@ async def listen_notifications(
         tasks = []
 
         async def handle_notification(notification: ChangeNotification) -> None:
-            callbacks = self.callbacks.get(type(notification), [fallback_callback])
+            callbacks = self.callbacks.get(type(notification), {fallback_callback})
             if not callbacks:
```

Comment:
After reading this again with fresh eyes, I'm wondering if it would make more sense to default to an empty list here, append the default callback if it's not None and then bail out here (maybe adjutsing the debug message to indicate that no callbacks nor default cb was given)?

Reply:
I'm not sure if the debug log needs any change, to be honest, since it never actually triggered before.
But otherwise changed.

Reply:
Looks like the changes did not get in with your recent push?

Reply:
Oops, forgot to amend the files, done now.